scripts/e2e-test/run-e2e-tests.js

```diff
@@ -0,0 +1,50 @@
+const { execSync, spawn } = require("child_process");
+const { join } = require("path");
+const { readFileSync, existsSync } = require("fs");
+
+const hasE2Etest = (packagePath) => {
+  const path = join(packagePath, "package.json");
+  if (!existsSync(path)) return false;
+  const pkgJson = JSON.parse(readFileSync(path).toString());
+  return Boolean(pkgJson.scripts["test:e2e"]);
+};
+
+const run = async () => {
+  /**
+   * Example output:
+   /path/to/package:@aws-sdk/client-accessanalyzer:1.0.0-gamma.3
+   /path/to/package:@aws-sdk/client-acm-pca:1.0.0-gamma.3
+   /path/to/package:@aws-sdk/client-acm:1.0.0-gamma.3
+   */
+  const changedPackagesRecord = execSync("npx lerna changed --all --parseable --long --loglevel silent");
```

Comment:
use lerna from package devDependency, so that:
* it doesn't have to be downloaded separately
* the same lerna version would be used while running e2e tests, and publishing

```suggestion
  const changedPackagesRecord = execSync("./node_modules/.bin/lerna changed --all --parseable --long --loglevel silent");
```

Reply:
addressed in f9a1fc0